### PR TITLE
Footsteps

### PR DESCRIPTION
**Pull request by theoajc**
- Footsteps sound added when player is grounded and has any velocity
- Pitch and volume changes based on speed
- Randomness in pitch and volume for realism

### CODE REVIEW

**Comment by theoajc** [comment]
Keep the player prefab and playermovement files- just renamed player to loudPlayer so as not to conflict

File: Assets/Scripts/FootstepsSounds.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        audioSource = gameObject.GetComponent<AudioSource> ();
+    }
+
+    // Update is called once per frame
+    void Update()
+    {   
+        //If player is grounded and moving then play sound
+        //High pitched sound for player running fast
+        //Random ranges of pitch and volume to add variation to step sounds
+        if (playerController.isGrounded && Input.GetKey("left shift") && !audioSource.isPlaying) {
```
Comment (JElgar) [inline Assets/Scripts/FootstepsSounds.cs:25]: This is great we should also use this for player movement logic

**Review by JElgar — APPROVED** [review]
LGTM. The only issue is the conflicts can you try and do the merge?

File: Assets/Scripts/FootstepsSounds.cs (hunk 1 above)
Comment (theoajc) [reply Assets/Scripts/FootstepsSounds.cs:23]: Happy to implement the movement tomorrow. 

Just tried to fix conflicts but just got really complicated so pushed from a new branch (footsteps3) but only the script and the audio- can update the prefab when we integrate.

**Pull request closed without merging**